### PR TITLE
Create a table for the Miro data

### PR DESCRIPTION
**Pull request by alexwlchan**

### CODE REVIEW

File: terraform/dynamo.tf [hunk 1]
```diff
@@ -30,16 +30,36 @@ resource "aws_dynamodb_table" "calm-dynamodb-table" {
...
     projection_type = "ALL"
   }
 
   global_secondary_index = {
     name            = "AltRefNo"
     hash_key        = "AltRefNo"
-    read_capacity   = 5
-    write_capacity  = 5
+    read_capacity   = 1
+    write_capacity  = 1
```
Comment (jamesgorrie) [inline terraform/dynamo.tf]: Is this really enough?

Comment (alexwlchan) [inline terraform/dynamo.tf]: Most of the time the tables aren’t in use, so we provision them to the lowest possible capacity and then our applications will provision more capacity as they need it. This just brings the default capacity into line with what the applications will set when they’re done.

See here: https://github.com/wellcometrust/platform-api/blob/master/calm_adapter/src/main/scala/uk/ac/wellcome/calm_adapter/modules/DynamoWarmupModule.scala

Comment (jamesgorrie) [inline terraform/dynamo.tf]: Ah, brilliant.

Comment (jamesgorrie) [inline terraform/dynamo.tf]: Can you set capacity to `0` then?

Comment (alexwlchan) [inline terraform/dynamo.tf]: Alas, 1 is the smallest capacity you can get.

**Pull request merged**